dnsapi/dns_cx.sh

```diff
@@ -82,7 +82,7 @@ existing_records() {
     return 1
   fi
   count=0
-  seg=$(printf "%s\n" "$response" | _egrep_o "{[^\{]*host\":\"$_sub_domain\"[^\}]*\}")
+  seg=$(printf "%s\n" "$response" | _egrep_o "{[^{]*host\":\"$_sub_domain\"[^}]*}")
```

Comment:
Hi,
I just fixed cx api, and tested on all the platforms.

Please rebase to the latest code and push again.

Thanks.